gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -701,6 +701,15 @@ function GM:PostCleanupMap()
 	ents.TTT.FixParentedPostCleanup()
 end
 
+---
+-- Called if CheckMapSwitch has determined that a map change should happen
```

Comment:
Some obligatory nitpicking:

The function mentioned here (line 705) is called "CheckForMapSwitch" and not "CheckMapSwitch"

Reply:
Done ._.